app/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,16 @@
         </activity>
         <activity android:name=".ui.home.MainActivity" />
         <activity android:name=".ui.auth.AuthActivity" />
+        <activity
+            android:name="org.fnives.test.showcase.compose.ComposeActivity"
+            android:configChanges="colorMode|density|fontScale|fontWeightAdjustment|keyboard|keyboardHidden|layoutDirection|locale|mcc|mnc|navigation|orientation|screenLayout|screenSize|smallestScreenSize|touchscreen|uiMode"
```

Comment:
This looks so weird 😂

Reply:
The idea is that when the activity is recreated the whole Compose UI is disposed and recreated, which is not needed because Compose can react to a config change and trigger a recomposition.